src/main/java/org/bonitasoft/connectors/rest/AbstractRESTConnectorImpl.java

```diff
@@ -202,14 +284,24 @@ protected final AuthorizationType getAuthType() {
     }
 
     protected final java.lang.String getProxyProtocol() {
+        if (getAutomaticProxyResolution()) {
```

Comment:
It will throw an NPE if `getAutomaticProxyResolution()` returns null. Use `Boolean.TRUE.equals` instead

Reply:
You should do the same for `getProxyHost` and `getProxyPort` as well